consensus/config/src/parameters.rs

```diff
@@ -131,12 +131,16 @@ impl Parameters {
         200
     }
 
-    pub(crate) fn default_sync_last_proposed_block_timeout() -> Duration {
-        Duration::ZERO
+    pub(crate) fn default_sync_last_known_own_block_timeout() -> Duration {
+        if cfg!(msim) {
+            Duration::from_millis(200)
+        } else {
+            Duration::from_secs(5)
```

Comment:
I believe that should be enough for a node with reasonable latencies to have reached and heard back from `f+1` nodes.

Reply:
Maybe leave a comment on we are preferring liveness to risk of equivocations here? In theory the timeout can be very long or infinite, with less risk of equivocations but more risk of stuck with liveness failures.